pagopa/io-app
# chore(Cross): [IOAPPX-354] Add the new `FooterActionsInline` component from `io-app-design-system`

## Short description
This PR adds the new `FooterActionsInline` component, to be used when you can't use the default `FooterActions` due to spacing or other reasons.

## List of changes proposed in this pull request
- Update `io-app-design-system` to `1.47.0`
- Add deprecation note to the local `FooterActions`
- Add specific screens in the DS section for demo purposes
- Replace local references of `FooterActions` in the DS section

### Preview

https://github.com/user-attachments/assets/9467230e-6ddb-4c38-8401-126e10ecb498



## How to test
Run the app in the local environment and check the new screens in the DS section
